charts/generic-service/templates/_envs.tpl

```diff
@@ -19,5 +19,9 @@ env:
   - name: {{ $key }}
     value: "{{ $val }}"
 {{- end }}
+{{- range $key, $val := .Values.env_comma_joined_from_list }}
```

Comment:
guess it is quite an edge case scenario, but note from line 7 that if you don't have any other environment variables then the `env:` block won't be output

Reply:
(And tested again to make sure it does indeed appear and disappear as expected)